crates/bevy_color/src/linear_rgba.rs

```diff
@@ -23,6 +23,22 @@ pub struct LinearRgba {
 impl StandardColor for LinearRgba {}
 
 impl LinearRgba {
+    /// A fully black color with full alpha.
+    pub const BLACK: Self = Self {
+        red: 0.0,
+        green: 0.0,
+        blue: 0.0,
+        alpha: 1.0,
+    };
+
+    /// A fully white color with full alpha.
+    pub const WHITE: Self = Self {
```

Comment:
It might also be useful to define the NONE color here as well, although I don't know what would use it.